include/render.h

```diff
@@ -152,6 +152,8 @@ struct RenderedImage {
 extern Render_t render;
 extern ScalerLineHandler_t RENDER_DrawLine;
 
+void RENDER_Reinit();
```

Comment:
The function is declared unconditionally, but only implemented when OpenGL is enabled, which trips minimal build. Probably needs more `#if C_OPENGL` sorcery?

Reply:
Ah yeah, pushed the changes, the forgot to check the CI status later. Definitely needs more ifdefs 😅

Reply:
Actually... it needed _less_ 😄